### PR TITLE
Add Packit

### PR DESCRIPTION
**Pull request by lzaoral**

### CODE REVIEW

**Comment by kdudka** [comment]
Nice work!  Merging...

File: .packit.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    - &copr
+      job: copr_build
+      trigger: commit
+      metadata:
+        targets:
+            - fedora-all-x86_64
+            - epel-all-x86_64
+        branch: main  # only used for commit trigger
+
+    - <<: *copr
```
Comment (TomasTomecek) [inline .packit.yaml:29]: wow, this is awesome, we should have this in our docs as a best practise

Comment (jamacku) [reply .packit.yaml:29]: @lzaoral Great workaround for https://github.com/packit/packit/issues/1440 :+1:

Comment (jamacku) [reply .packit.yaml:29]: @TomasTomecek I have created PR for docs, Please have a look:
- https://github.com/packit/packit.dev/pull/540

**Pull request merged**